src/pages/instances/EditInstance.tsx

```diff
@@ -112,32 +113,41 @@ const EditInstance: FC<Props> = ({ instance }) => {
       // ensure the etag is set (it is missing on the yaml)
       instancePayload.etag = instance.etag;
 
-      void updateInstance(instancePayload, project).then((operation) => {
-        const instanceName = values.name;
-        const instanceLink = instanceLinkFromName({
-          instanceName,
-          project,
+      void updateInstance(instancePayload, project)
+        .then((operation) => {
+          const instanceName = values.name;
+          const instanceLink = instanceLinkFromName({
```

Comment:
we could move this outside of the then block and reuse the variable in the catch

Reply:
In the catch block we don't have an operation at hand. The op comes from the response to the request that failed in the catch case.

Reply:
Oh wait, this is not using the operation, but the name, yeah -- that makes sense what you said now :)